variables.tf

```diff
@@ -94,13 +69,14 @@ variable "default_node_groups" {
     {
       compute = object(
         {
-          ami                  = optional(string, null)
-          bootstrap_extra_args = optional(string, "")
-          instance_types       = optional(list(string), ["m5.2xlarge"])
-          spot                 = optional(bool, false)
-          min_per_az           = optional(number, 0)
-          max_per_az           = optional(number, 10)
-          desired_per_az       = optional(number, 1)
+          ami                   = optional(string, null)
+          bootstrap_extra_args  = optional(string, "")
+          instance_types        = optional(list(string), ["m5.2xlarge"])
+          spot                  = optional(bool, false)
+          min_per_az            = optional(number, 0)
+          max_per_az            = optional(number, 10)
+          desired_per_az        = optional(number, 1)
+          availability_zone_ids = optional(list(number), [1, 2, 3])
```

Comment:
this isn't an az zone id...
it should be usw2-az4, no one will understand 1, 2, 3.  Its not what the API returns.

Reply:
pushed ff97d6762c554b843e019dcba2ea9b42807c7bd2.